_docs-2/administration/upgrading.md

```diff
@@ -30,12 +30,18 @@ Below are some changes in 2.0 that you should be aware of:
    - Tablet server hosts must be listed in a `tservers` file instead of a `slaves` file. To minimize confusion, Accumulo will not start if the old `slaves` file is present.
 * `accumulo-service` script can be used to start/stop Accumulo services (i.e master, tablet server, monitor) on a single node.
     - Can be used even if Accumulo was started using `accumulo-cluster` script.
-* Accumulo 2.0 logging is configured by three files. Any previous logging customization should be added to these files.
+* `accumulo-env.sh` constructs environment variables (such as `JAVA_OPTS` and `CLASSPATH`) used when running Accumulo processes
+    - This file was used in Accumulo 1.x but has changed signficantly for 2.0
+    - Environment variables (such as `$cmd`, `$bin`, `$conf`) are set before `accumulo-env.sh` is loaded and can be used to customize environment.
+    - The `JAVA_OPTS` variable can be customized to pass command-line arguments to the `java` command that starts Accumulo processes.
```

Comment:
This is good... but in my previous review, I used a specific wording, because it's important to understand that `JAVA_OPTS` is special and distinct from the others.

I'm not asking you to use my wording, but I do think it's important to understand that `JAVA_OPTS` is just something we made... and we're explicitly using, whereas things like `CLASSPATH` (and the `MALLOC_*` thing) are not used by us directly, but by other parts of the system. The only thing *we* are specifically asking for (the expected "output" of `accumulo-env.sh`) is `JAVA_OPTS` (although `CLASSPATH` should be set either in the script or before the script is run, otherwise, we won't get very far.

I'm not sure this gives enough attention to `JAVA_OPTS`, since it's really the reason why we're running this script (other exported env, or temporary variables aren't required "output" from this script).

Reply:
I updated it again with 29a69f0ab4ea01fb2e3ad58520c

does that work?